packages/services-store/src/actions/__tests__/profileAPI.test.ts

```diff
@@ -50,6 +56,91 @@ describe('Redux actions for `ProfileAPI`', () => {
     ]);
   });
 
+  it('dispatches the action of error in monitoring user authentication status', async () => {
+    ProfileAPI.getUserStatus.mockRejectedValue(new Error('error-getuserstatus'));
```

Comment:
I think you'll also have to mock the other ProfileAPI methods for Cloud cookie and Docs API here since `loadUserStatus(MASTHEAD_AUTH_METHOD.COOKIE)` and `loadUserStatus(MASTHEAD_AUTH_METHOD.DOCS_API)` will call the `ProfileAPI.checkCloudCookie();` and `ProfileAPI.checkCloudDocsAPI();` methods respectively. 
```suggestion
    ProfileAPI.getUserStatus.mockRejectedValue(new Error('error-getuserstatus'));
    ProfileAPI.checkCloudCookie.mockRejectedValue(new Error('error-getuserstatus'));
    ProfileAPI.checkCloudDocsAPI.mockRejectedValue(new Error('error-getuserstatus'));
```